openshift-knative-operator/cmd/operator/kodata/knative-eventing/1.11/2-eventing-core.yaml

```diff
@@ -5033,3 +5033,27 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
   name: openshift-serverless-view-eventing-configmaps
+---
+# Copyright 2020 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: config-openshift-trusted-cabundle
+  namespace: knative-eventing
+  labels:
```

Comment:
I don't know the context here, just to not miss it. @pierDipi shouldn't we have a specific label to filter on CMs to trust for? Or is this here not related?

Reply:
here is the context: https://github.com/openshift-knative/eventing/commit/d8390aacd94efe867480928d4ec8fd41c3d9c6dc

Reply:
This is named towards `openshift` and things like volume/source will reflect that on their names as well. It will not conflict w/ things from future trust-manager / kube 1.27